internal/xds/translator/tracing.go

```diff
@@ -132,7 +132,7 @@ func processClusterForTracing(tCtx *types.ResourceVersionTable, tracing *ir.Trac
 		name:         clusterName,
 		settings:     []*ir.DestinationSetting{ds},
 		tSocket:      nil,
-		endpointType: DefaultEndpointType,
+		endpointType: EndpointTypeDNS,
```

Comment:
is this always DNS ?

Reply:
The previous default one is DNS, but the name is unclear, so I changed the name.

Reply:
cc @zirain who may have more context

Reply:
thanks, it's more clear now.